[2024-09-14T20:54:28.572+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-14T20:54:28.650+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbyte_sync_tables_postgres.airbyte_trigger_sync manual__2024-09-14T20:54:24.497914+00:00 [queued]>
[2024-09-14T20:54:28.676+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbyte_sync_tables_postgres.airbyte_trigger_sync manual__2024-09-14T20:54:24.497914+00:00 [queued]>
[2024-09-14T20:54:28.678+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-14T20:54:28.729+0000] {taskinstance.py:2330} INFO - Executing <Task(AirbyteTriggerSyncOperator): airbyte_trigger_sync> on 2024-09-14 20:54:24.497914+00:00
[2024-09-14T20:54:28.753+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'airbyte_sync_tables_postgres', 'airbyte_trigger_sync', 'manual__2024-09-14T20:54:24.497914+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airbyte_sync_tables_postgres.py', '--cfg-path', '/tmp/tmp2g8i4l07']
[2024-09-14T20:54:28.762+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask airbyte_trigger_sync
[2024-09-14T20:54:28.806+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=372) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-14T20:54:28.809+0000] {standard_task_runner.py:64} INFO - Started process 373 to run task
[2024-09-14T20:54:28.919+0000] {task_command.py:426} INFO - Running <TaskInstance: airbyte_sync_tables_postgres.airbyte_trigger_sync manual__2024-09-14T20:54:24.497914+00:00 [running]> on host e532e2e16082
[2024-09-14T20:54:29.084+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vizentec-engdados' AIRFLOW_CTX_DAG_ID='airbyte_sync_tables_postgres' AIRFLOW_CTX_TASK_ID='airbyte_trigger_sync' AIRFLOW_CTX_EXECUTION_DATE='2024-09-14T20:54:24.497914+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-14T20:54:24.497914+00:00'
[2024-09-14T20:54:29.087+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-14T20:54:29.133+0000] {base.py:84} INFO - Using connection ID 'airbyte_conn' for task execution.
[2024-09-14T20:54:30.050+0000] {airbyte.py:95} INFO - Job 16 was submitted to Airbyte Server
[2024-09-14T20:54:30.052+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-14T20:54:30.089+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=airbyte_sync_tables_postgres, task_id=airbyte_trigger_sync, run_id=manual__2024-09-14T20:54:24.497914+00:00, execution_date=20240914T205424, start_date=20240914T205428, end_date=20240914T205430
[2024-09-14T20:54:30.153+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-14T20:54:30.195+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-14T20:54:30.201+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
